src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -14,27 +14,145 @@
 limitations under the License.
 -->
 
-<div layout="column" layout-margin layout-align="center center">
-  Displaying details for replica set {{ctrl.replicaSetDetail.name}} in namespace
-  {{ctrl.replicaSetDetail.namespace}}.
-  <div>
-    <table>
-      <thead>
-        <tr>
-          <th>Pod</th>
-          <th>Start time</th>
-          <th>IP</th>
-          <th>Node</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
-          <td>{{pod.name}}</td>
-          <td>{{pod.startTime}}</td>
-          <td>{{pod.podIP}}</td>
-          <td>{{pod.nodeName}}</td>
-        </tr>
-      </tbody>
-    </table>
+<md-content layout="row" layout-fill flex>
+  <div class="kd-replicasetdetail-sidebar" layout="column">
+    <div flex layout-align="start center" layout="row" class="kd-header-option-panel">
+      <a flex-offset="10" ui-sref="replicasets">
+        <i class="material-icons kd-align-mid kd-font-lg">keyboard_backspace</i>
+      </a>
+      <span flex-offset="5" flex class="kd-align-mid md-title">
+        {{ctrl.replicaSetDetail.name}}
+      </span>
+    </div>
+    <div flex-offset="5" flex layout-align="start start" layout-padding layout="row">
+      <div flex layout="column">
+        <h3 layout="row" class="kd-primary md-body-2">
+          <div flex="55">
+            <i class="kd-align-mid material-icons">refresh</i>
+            <span class="kd-align-mid">ROLLING UPDATE</span>
+          </div>
+          <div flex-offset="5" flex>
+            <i class="kd-align-mid material-icons">delete</i>
+            <span class="kd-align-mid">DELETE</span>
+          </div>
+        </h3>
+        <h1 class="md-body-2">
+            <span class="kd-align-mid">
+              {{ctrl.replicaSetDetail.pods.length}} pods
+            </span>
+          <i class="kd-editpen-md material-icons kd-primary kd-align-mid">
+            mode_edit
+          </i>
+        </h1>
+        <h1 class="md-caption kd-font-1">Label selector</h1>
+        <h1 class="md-caption kd-font-2">
+          <b>{{ctrl.formatLabelString(ctrl.replicaSetDetail.labelSelector)}}</b>
+        </h1>
+        <h1 class="md-body-2">Replica Sets</h1>
+        <h1 class="md-caption kd-font-1">Labels</h1>
```

Comment:
The reason why i'm using headers(h1, h2, etc) is that they have padding(top, bottom) added by default to them and combined with `md-body` `md-caption` i don't have to write css to add vertical spacing myself.

Also I was thinking about font coloring because I've done investigation if this can be tied to mdThemingProvider and it seems that there is no option to do that now. There were few workarounds for that, but they involve some custom services. `kd-font-1` class is in actually in `my-detail` css file but to have consistent font colors across project we could extract that somewhere higher.